node/src/GlideClusterClient.ts

```diff
@@ -659,6 +661,67 @@ export class GlideClusterClient extends BaseClient {
         );
     }
 
+    /**
+     * Invokes a previously loaded function.
+     *
+     * See https://valkey.io/commands/fcall/ for more details.
+     *
+     * since Valkey version 7.0.0.
+     *
+     * @param func - The function name.
+     * @param args - A list of `function` arguments and it should not represent names of keys.
+     * @param route - The command will be routed to a random node, unless `route` is provided, in which
+     *  case the client will route the command to the nodes defined by `route`.
+     * @returns The invoked function's return value.
+     *
+     * @example
+     * ```typescript
+     * const response = await client.fcallRoute("Deep_Thought", undefined, "randomNode");
+     * console.log(response); // Output: Returns the function's return value.
+     * ```
+     */
+    public fcallRoute(
```

Comment:
Does it work if you name it just `fcall`? If not, I think `fcallWithRoute` is a better name.
Same applicable to fcall_ro